sale_order_line_variant_description/tests/test_sale_order_line_variant_description.py

```diff
@@ -1,24 +1,24 @@
-# Copyright 2015-17 Alex Comba - Agile Business Group
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
I think modifying existing copyright headers is not expected in migration commits.